src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/operatorwrap/InputOperatorWrapSeparatorAndInterfaces.java

```diff
@@ -1,6 +1,6 @@
-package com.puppycrawl.tools.checkstyle.checks.whitespace;
+package com.puppycrawl.tools.checkstyle.checks.whitespace.operatorwrap;
```

Comment:
Let's not do few things in such complicated updates. We just move files in subfolders. Do not change structure of them or ... .
All extra action and optimization should be done later and separate commits and PRs.